src/main/java/backend/UpdateController.java

```diff
@@ -107,32 +111,37 @@ public void processAndRefresh(List<FilterPanel> filterPanels) {
     }
 
     /**
-     * Filters, sorts and counts issues within the model according to the given filter expressions.
+     * Filters, sorts and counts issues within the model according to the given panels' filter expressions.
      * In here, "processed" is equivalent to "filtered, sorted and counted".
      *
-     * @param filterExprs Filter expressions to process.
+     * @param filterPanels Filter panels to process.
      * @return Filter expressions and their corresponding issues after filtering, sorting and counting.
      */
-    private Map<FilterExpression, List<GuiElement>> processFilter(List<FilterExpression> filterExprs) {
+    private Map<FilterExpression, List<GuiElement>> processPanels(List<FilterPanel> filterPanels) {
```

Comment:
It is possible for a panel to have a erroneous filter which is not applied yet, when the user just types without confirming (at least that's the behaviour for now).
In that case, applying the same erroneous filter in another panel will cause a change in both panels, which might not be appropriate.